packages/web/src/components/play-bar/playback-rate-button/PlaybackRateButton.tsx

```diff
@@ -5,15 +5,15 @@ import { PopupMenu, PopupMenuItem, PopupPosition } from '@audius/stems'
 import cn from 'classnames'
 import { useDispatch, useSelector } from 'react-redux'
 
-import { ReactComponent as Icon0_5x } from 'assets/img/iconPlaybackRate0_5x.svg'
-import { ReactComponent as Icon0_8x } from 'assets/img/iconPlaybackRate0_8x.svg'
-import { ReactComponent as Icon1_1x } from 'assets/img/iconPlaybackRate1_1x.svg'
-import { ReactComponent as Icon1_2x } from 'assets/img/iconPlaybackRate1_2x.svg'
-import { ReactComponent as Icon1_5x } from 'assets/img/iconPlaybackRate1_5x.svg'
-import { ReactComponent as Icon1x } from 'assets/img/iconPlaybackRate1x.svg'
-import { ReactComponent as Icon2_5x } from 'assets/img/iconPlaybackRate2_5x.svg'
-import { ReactComponent as Icon2x } from 'assets/img/iconPlaybackRate2x.svg'
-import { ReactComponent as Icon3x } from 'assets/img/iconPlaybackRate3x.svg'
+import Icon0_5x from 'assets/img/iconPlaybackRate0_5x.svg'
+import Icon0_8x from 'assets/img/iconPlaybackRate0_8x.svg'
+import Icon1_1x from 'assets/img/iconPlaybackRate1_1x.svg'
+import Icon1_2x from 'assets/img/iconPlaybackRate1_2x.svg'
```

Comment:
Just a call out that we should definitely migrate all the stuff to harmony at some point

Reply:
Down